src/ripple/app/tx/impl/Transactor.h

```diff
@@ -82,8 +83,20 @@ struct PreclaimContext
 class TxConsequences;
 struct PreflightResult;
 
+/** Represents the entry in a singer list */
+using SignerEntry = std::pair<AccountID, std::uint16_t>;
```

Comment:
A middle ground, if you would like to keep the meaningful field names, is to just remove the constructor from the old `struct`, treat it as an aggregate, and define `operator<` for it.